cirq/devices/device.py

```diff
@@ -105,3 +142,27 @@ def can_add_operation_into_moment(
             Whether or not the moment will validate after adding the operation.
         """
         return not moment.operates_on(operation.qubits)
+
+
+@value.value_equality
+class QidPair:
```

Comment:
do we need this class? A tuple would do just fine

Reply:
I guess this is good in the case where there's no directionality to the qubit pair, which again is an assumption about how the device works :)

Reply:
This class would be a lot more usable if it supported unpacking, maybe by sorting the qubits first.

```python
qa, qb = pair
```

Reply:
done